docs/user/machine-api-operator-overview.md

```diff
@@ -0,0 +1,82 @@
+# Document Purpose
+
+This document describes how the  Machine API Operator (herein `MAO`) gets deployed, and the `MAO`'s responsibilities. It includes guidelines on how to troubleshoot typical issues with resources provisioned by `MAO`.
+
+## CVO integration
+
+`MAO` code is running under the `machine-api-operator` `Deployment`. This and other resources listed below rely on the `CVO` management strategy. These resources are consumed from the [`/install` directory]( https://github.com/openshift/machine-api-operator/tree/master/install).
+
+Namely:
+- `openshift-machine-api` `Namespace` - the primary and the only namespace, where the `MAO` is hosted, and all of the dependent resources are being managed by the operator.
+- `machine-api-operator` `Deployment` - `MAO` code runs here.
+- `machine-api-operator-images` `ConfigMap` - is injected with provider images, used by `MAO` to deploy cloud-provider specific code.
```

Comment:
TODO: add a section with a brief explanation about how this data is injected and who generates those images (eg, release tooling).  Too much info for this line, doesn't need to be this patch set.

Reply:
I'll add a comment in the doc, so we could return to this question later.

Reply:
@michaelgugino  Realized the comment is not what was needed, so there is a breif overview of `ImageStreams` we use.